src/stylesheets/common/_helpers.scss

```diff
@@ -172,14 +172,13 @@
   color: #333333;
 }
 
-.text-16 {
-  // font-weight: normal;
-  // font-size: 16px;
-  // line-height: 1.6;
+.deemphasize-text {
```

Comment:
Oh right.  I'm even the one who used `text-muted` on that page.  We probably need to keep the two separate colors (#777 and #ccc are pretty far apart), but I can change this class name to match.  Good call.